drivers/fpga/Kconfig.altera

```diff
@@ -0,0 +1,10 @@
+# FPGA ALTERA driver configuration options
+
+# Copyright (c) 2024, Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+config ALTERA_FPGA
```

Comment:
Since this is only one type of interface (agilex bridge) it might be best to rename this option, because. Otherwise, it might be mistaken as support for all altera fpgas